catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -366,32 +324,33 @@ private boolean isMyRoute(String routeId) {
   private RouteBuilder createRouteBuilder() {
     return new RouteBuilder() {
       @Override
-      public void configure() throws Exception {
+      public void configure() {
         StringBuilder stringBuilder = new StringBuilder();
         // Configure the camel route to ignore changing files (larger files that are in the process
         // of being copied)
         // Set the readLockTimeout to 2 * readLockIntervalMilliseconds
         // Set the readLockCheckInterval to check every readLockIntervalMilliseconds
-        boolean isDav = false;
-
-        if (monitoredDirectory.startsWith("http")) {
-          isDav = true;
-        } else {
-          stringBuilder.append("file:" + monitoredDirectory);
-          stringBuilder.append("?recursive=true");
-          stringBuilder.append("&moveFailed=.errors");
-
-          /* ReadLock Configuration */
-          stringBuilder.append("&readLockMinLength=1");
-          stringBuilder.append("&readLock=changed");
-          stringBuilder.append("&readLockTimeout=" + (2 * readLockIntervalMilliseconds));
-          stringBuilder.append("&readLockCheckInterval=" + readLockIntervalMilliseconds);
-
-          /* File Exclusions */
-          String exclusions = getBlackListAsRegex();
-          if (StringUtils.isNotBlank(exclusions)) {
-            stringBuilder.append("&exclude=" + exclusions);
-          }
+
+        stringBuilder.append("file:");
```

Comment:
✏️ These blocks could be chained. e.g.
```java
stringBuilder
  .append("something")
  .append("other")
  .append(things);
```
or whatever google-format would do to that.